essays/AITBW-roadToDune/index.md

```diff
@@ -0,0 +1,83 @@
+# Dr. Strangelove or: How I handled my stubbornness to become a web developer
+
+Becoming a developer can be a pretty underwhelming task these days, especially if you delve into the [JavaScript](https://hackernoon.com/how-it-feels-to-learn-javascript-in-2016-d3a717dd577f) land, where _the new kid on the block_ can be thrown into obscurity in a matter of days.
+
+I faced a completely different situation back when I began college in 2010, which is the story I'd like to tell you today.
+
+I enrolled in college around mid/late 2010 under an "Information Engineer" undergrad, which is mostly focused in the importance/value of data and how we can take advantage of IT systems to make the most out of it.
+
+When I began college, one of my biggest weaknesses from high school were maths. My education had a very high standard but I just couldn't make sense out of them —as a matter of fact: I struggled with maths during **all** of my high school education.
+
+You must be asking: Then why did you went for Engineering?
+
+1. I love a good challenge
+2. I didn't see myself in the future as anything other than an Engineer
```

Comment:
Do you mind elaborating on why? This might be something relevant for the reader. Many people I know were influenced into becoming engineers because their parents were engineers. Was that the case for you?

Reply:
Hmm 🤔 There are several points to elaborate further on this one:

* My dad pushed me into the IT world from a very early age
* I used to be an avid gamer, so PCs were fascinating for me; especially selecting the components to build one from start
* Engineering just seemed the right choice for me even if I wasn't very good with maths

Reply:
Please update the markdown 🤗